test/integration/es7.test.js

```diff
@@ -1,14 +1,14 @@
-let _ = require('lodash/fp')
-let Contexture = require('contexture')
-// let provider = require('contexture-elasticsearch')
-// let types = require('contexture-elasticsearch/types')
-let Provider = require('../../src/index')
-let types = require('../../src/types')
-let elasticsearch = require('@elastic/elasticsearch')
-let AgentKeepAlive = require('agentkeepalive')
+import _ from 'lodash/fp.js'
+import Contexture from 'contexture'
+// import provider from 'contexture-elasticsearch.js'
```

Comment:
This is commented out, but I don't tink this is the right import since `contexture-elasticsearch` is a module